.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.yarnpkg.com
```

Comment:
Where does that file come from, is it needed? Same for `.yarnrc`

Reply:
We use a private repo at our company, which would result in links to our private registry ending up in the yarn.lock file. This instructs yarn/npm to always use that repository. I can also remove it, but in my experience it's a really nice safeguard to ensure that you don't accidentally end up with wrong links.